rs/execution_environment/src/execution_environment/tests.rs

```diff
@@ -291,21 +291,19 @@ fn output_requests_on_system_subnet_ignore_memory_limits() {
         .with_manual_execution()
         .build();
 
-    let canister_id = test.create_canister(Cycles::new(1_000_000_000));
-    test.install_canister_with_allocation(
-        canister_id,
-        wat::parse_str(CALL_SIMPLE_WAT).unwrap(),
-        None,
-        Some(canister_memory as u64),
-    )
-    .unwrap();
+    let canister_id = test
+        .create_canister_with_allocation(
+            Cycles::new(1_000_000_000),
+            None,
+            Some(canister_memory as u64),
+        )
+        .unwrap();
+    test.install_canister(canister_id, wat::parse_str(CALL_SIMPLE_WAT).unwrap())
+        .unwrap();
     test.ingress_raw(canister_id, "test", vec![]);
     test.execute_message(canister_id);
 
-    assert_eq!(
-        test.subnet_available_memory().get_execution_memory(),
-        (size_of::<CanisterChange>() + size_of::<PrincipalId>()) as i64
-    );
+    assert_eq!(test.subnet_available_memory().get_execution_memory(), 0);
```

Comment:
Why this drop? (the original values looks like the size of one canister history entry)

Reply:
I'm not quite sure, I forgot to come back to this. It seems accurate to me that the current status is like it is now. The subnet's execution memory is set to `canister_memory` in the beginning and the canister requires all of it as memory allocation. So the available should be 0. But it's puzzling why the previous version was working...

Reply:
Discussed offline.